src/scss/pages/_about.css

```diff
@@ -0,0 +1,43 @@
+.aboutcontainer {
+  display: flex;
+}
+.spaceship {
+  max-width: 400px;
+  max-height: 400px;
+}
+.leftimage {
+  float: left;
+  width: 40%;
+}
+.aboutcard {
+  font-size: 25px;
+  width: 70%;
+  margin-right: 5%;
+}
+.aboutheader h3 {
+  margin-left: 40%;
+  margin-top: 10%;
+}
+.aboutcontent {
+  padding: 10%;
+  padding-top: 4%;
+}
+.para2 {
+  margin-top: 4%;
+}
+.Recpurple {
+  width: 10%;
+  transform: translate(-18em, -23em);
```

Comment:
use absolute position to position it, as translate will cause problems with the rotate animations

Reply:
transform is a better a way to do this.